### PR TITLE
closes #196

### LINKED ISSUE #196: Differentiate columns with no annotations and ignored columns
There is no difference between the columns which were not disambiguate because they were ignored by default (e.g. becase they were not considered as named entities) and the columns which were disambiguated, but no disambiguation found. 

E.g. below, the district code has no annotations because it is not a named entity column, but the NUTS2 column has no annotations because I explicitely removed disambiguations, and NUTS3 has no annotations because there were no disambiguations in the target KB

screen shot 2016-12-15 at 17 14 07

Solution: Emphasize somehow which column is considered as non-named entity and ignored or in more general emphasize somehow the ignored columns to differentiate them from columns for which there is no annotation.

### PR DESCRIPTION
**Pull request by KataBoku**

### CODE REVIEW

**Comment by tomas-knap** [comment]
Cannot be merged so far, lets discuss during the call shortly

**Comment by brodecva** [comment]
@KataBoku Hi, as discussed during the last call with @tomas-knap , use only the types from the primary base in the tooltip (and leave the KB name).

File: src/templates/taskresult/view/classdisam/cdtable/cdtablerow/cdtablerow.js [hunk 1]
```diff
@@ -37,19 +37,44 @@
...
+                //returns title with information about  the success of algorithm classification
+                scope.getTitle = function (columnIndex) {
+                    if (scope.rowIndex == -1) {
+                        var types = scope.columnTypes[columnIndex].processingType;
+                        var title = [];
+                        for (var kb in types) {
+                            title.push("Values of ",kb);
+                            switch (types[kb]) {
+                                case "IGNORED":
+                                    title.push(" were ignored.\n");
```
Comment (brodecva) [inline src/templates/taskresult/view/classdisam/cdtable/cdtablerow/cdtablerow.js]: Rather: "Ignored column."

File: src/templates/taskresult/view/classdisam/cdtable/cdtablerow/cdtablerow.js [hunk 2]
```diff
@@ -37,19 +37,44 @@
...
+                                    break;
+                                case "NAMED_ENTITY":
+                                    title.push(" were determined.\n");
```
Comment (brodecva) [inline src/templates/taskresult/view/classdisam/cdtable/cdtablerow/cdtablerow.js]: Rather: "Named entity"

File: src/templates/taskresult/view/classdisam/cdtable/cdtablerow/cdtablerow.js [hunk 3]
```diff
@@ -37,19 +37,44 @@
...
+                                    break;
+                                case "NON_NAMED_ENTITY":
+                                    title.push(" cannot be determined.\n");
```
Comment (brodecva) [inline src/templates/taskresult/view/classdisam/cdtable/cdtablerow/cdtablerow.js]: Rather: "Non-named entity"

File: src/templates/taskresult/view/classdisam/cdtable/cdtablerow/cdtablerow.js [hunk 4]
```diff
@@ -37,19 +37,44 @@
...
             },
             templateUrl: currentFolder + 'cdtablerow.html',
             link: function (scope, iElement, iAttrs) {
 
+                //returns title with information about  the success of algorithm classification
+                scope.getTitle = function (columnIndex) {
+                    if (scope.rowIndex == -1) {
+                        var types = scope.columnTypes[columnIndex].processingType;
+                        var title = [];
+                        for (var kb in types) {
```
Comment (brodecva) [inline src/templates/taskresult/view/classdisam/cdtable/cdtablerow/cdtablerow.js]: As discussed, use only the primary base.

**Pull request merged**